adapters/postgres.go

```diff
@@ -479,48 +485,77 @@ func (p *Postgres) BulkInsert(table *Table, objects []map[string]interface{}) er
 }
 
 func (p *Postgres) insertInTransaction(wrappedTx *Transaction, table *Table, objects []map[string]interface{}) error {
```

Comment:
It would be better to rename this func to batchInsertInTransaction or bulkInsertInTransaction

Reply:
✅